bug_report.php

```diff
@@ -302,6 +303,29 @@
 
 layout_page_header_begin();
 
+if ( ( ON === config_get_global( 'public_urls' ) ) && isset( $t_bug_data->token ) && ( !is_null( $t_bug_data->token ) ) ) {	
```

Comment:
@vboctor 

I was not sure which file I should use for that function. Do you have any suggestions?